packages/manager/modules/nutanix/src/dashboard/general-info/module.js

```diff
@@ -17,6 +19,8 @@ angular
     'ui.router',
     ovhManagerBillingComponents,
     serverSupport,
+    region,
+    editName,
   ])
   .config(routing)
   .component('nutanixGeneralInfo', component)
```

Comment:
```suggestion
  .component('nutanixGeneralInfoComponent', component)
```

Reply:
Change not related to this PR. Need to do a refactoring to make this change. It need to be done as part of a new PR.